CPS-0004/README.md

```diff
@@ -0,0 +1,96 @@
+4. This system should not undermine it's ability to serve it's financial function, such as overburdening the chain, damaging the monetary policy, etc.
+5. This system should be implementable within a reasonable time frame and budget.
+6. This system should be recursively updatable; if improvements or changes to the system of governance are needed, they should be voted on and enacted via this system.
+7. This system is highly dependent on the off chain user experience and community tooling, since very few users are reading raw CBOR dumps from the chain.
+8. This system should be cognizant of the fact that different decisions can have different impact and risk thresholds.
+9. This system cannot avoid the fact that stake pool operators of the network, collectively, always maintain control over a "pocket nuke": under sufficient discontent and misalignment with the will of governance, they can modify the code and initiate an uncontrolled hard fork to produce a separate chain aligned with their values.
+
+Additionally, here is a more opinionated (and potentially controversial) list of potential considerations for a CIP, presented purely to foster discussion:
+
+1. There is currently no known decentralized "proof of personhood" system that is well specified and easily implementable, and without such a system, "one lovelace one vote" systems are the most effective defense against sybil attacks.
+2. Given the current near total control over these decisions that these founding powers have, any dilution of power that doesn't compromise the security of the chain or risk a deadlock is a worthwhile step to take.
+3. The system loses legitimacy if participation is extremely low, or participation is severely uninformed; If a significant portion of the Cardano community has not yet had a reasonable opportunity to participate, or the majority of participation is indinstinguishable from randomness, then any decisions made during this time are illegitimate.
+4. Not every ADA holder is interested in *directly* participating in every decision. Often an ADA holders voice can still be heard through delegation to another ADA holder.
+5. If the system contains such a notion of delegation, the ability to withdraw, redirect, or reclaim that voice at any time is an important tool in fighting corruption.
+6. While it may be useful to draw inspiration from existing real-world systems, it should also be understood that blockchain governance will have an entirely different set of constraints.  For example, the ADA community does not share one culture or geographic locality. Additionally, the decisions being made are not the same kinds of decisions that a world government needs to make.
+7. The uncontrolled hard fork described above is dangerous (from a safety standpoint) and disruptive (from an economic standpoint); it will likely seriously undermine the cohesion of the Cardano community. For this reason, it should be considered a tool of absolute last resort.
```

Comment:
```suggestion
7. This system cannot rely on a trusted third party for "proof of personhood".  There is currently no known decentralized  system that is well specified and easily implementable, and without such a system, "one lovelace one vote" systems are the most effective defense against sybil attacks.
8. This system should be able to ensure the legitimacy of decisions. It loses legitimacy if participation is extremely low, or participation is severely uninformed; If a significant portion of the Cardano community has not yet had a reasonable opportunity to participate, or the majority of participation is indinstinguishable from randomness, then any decisions made during this time are illegitimate.

Additionally, here is a more opinionated (and potentially controversial) list of potential considerations for a CIP, presented purely to foster discussion:

1. This system is highly dependent on the off chain user experience and community tooling, since very few users are reading raw CBOR dumps from the chain.
2. This system should be cognizant of the fact that different decisions can have different impact and risk thresholds.
3. This system cannot avoid the fact that stake pool operators of the network, collectively, always maintain control over a "pocket nuke": under sufficient discontent and misalignment with the will of governance, they can modify the code and initiate an uncontrolled hard fork to produce a separate chain aligned with their values.
4. Given the current near total control over these decisions that these founding powers have, any dilution of power that doesn't compromise the security of the chain or risk a deadlock is a worthwhile step to take.
5. Not every ADA holder is interested in *directly* participating in every decision. Often an ADA holders voice can still be heard through delegation to another ADA holder.
6. If the system contains such a notion of delegation, the ability to withdraw, redirect, or reclaim that voice at any time is an important tool in fighting corruption.
7. While it may be useful to draw inspiration from existing real-world systems, it should also be understood that blockchain governance will have an entirely different set of constraints.  For example, the ADA community does not share one culture or geographic locality. Additionally, the decisions being made are not the same kinds of decisions that a world government needs to make.
8. The uncontrolled hard fork described above is dangerous (from a safety standpoint) and disruptive (from an economic standpoint); it will likely seriously undermine the cohesion of the Cardano community. For this reason, it should be considered a tool of absolute last resort.
```

Reply:
Starting point 7 in Goals the points became more like "notes" which belong to the next category imo.
The point about proof of identity and the legitimacy should be stated as goals, not just notes. This is especially because these are the two points causing the most controversy, and debates should start from the same set of agreed upon goals. 

For example, solving decentralized "proof of personhood" is a long standing computer science problem and probably not going to be solved by the Cardano community (no offense). 

I like how you phrased the legitimacy problem, it gives us a good start to define "legitimacy of a decision".

Reply:
Sorry again for the large diff, it seems inevitable when moving lines around, even if the content is very similar

Reply:
The split isn't intended to be between "goals" and "notes".

Instead, all of them are goals, constraints, considerations, etc.

And then they're split between those that I felt we're fairly uncontroversial (i.e. it's hard to imagine arguing that governance should undermine the security of the protocol), and those that still seem unsettled within the community, regardless of what my own opinions are.  For example, you and I feel that proof of personhood is likely unsolvable, but others seem unconvinced. So I'm simple being honest about how much debate there is over the point. :)

Reply:
Ok I see, do you think there should be a "things we know for sure and that form the basis for CIP1694" list? Which is what I thought the "goals" section was, and why I edited it in this way.